fix color scheme

cedrics opened the pull request:
I went through the variables and there were some changes introduced by me on accident. This should fix them all and close #606

adrianmroz commented:
Something happened also here: https://github.com/remerge/turnilo/blob/2528cdffd276d7bd2a9f3590668d843ae988d1fe/src/client/utils/styles/_header-bar.scss#L91

This is #604 - results in white on white text. Could you check?

cedrics commented:
> Something happened also here: https://github.com/remerge/turnilo/blob/2528cdffd276d7bd2a9f3590668d843ae988d1fe/src/client/utils/styles/_header-bar.scss#L91
> 
> This is #604 - results in white on white text. Could you check?

Done, was a bit too eager and replaced the wrong variable. Reverted that change

adrianmroz commented:
Let's wait for travis and I'll merge. With my fix for split limit we have another beta version :)
